measures/BuildExistingModel/measure.rb

```diff
@@ -28,7 +28,7 @@ def name
 
   # human readable description
   def description
-    return 'Builds the OpenStudio Model for an existing building.'
+    return 'Builds the OpenStudio Model for a existing building.'
```

Comment:
Why are we replacing proper grammar with improper grammar?

Reply:
We isn't. It's a temporary change to see if results_output.csv changes. :)